packages/create-pantheon-decoupled-kit/src/templates/tailwindless-next/pages/examples/auth-api.module.css

```diff
@@ -0,0 +1,35 @@
+.container {
+  max-width: var(--md);
+  margin: var(--10) auto 0;
+}
+
+.container > h1 {
+  margin-bottom: var(--4);
+  font-size: var(--12);
+  font-weight: 800;
+}
+
+.container > [href] {
+  text-decoration: underline;
+  font-size: var(--5);
+  font-weight: 500;
+}
+
+.container > div {
+  margin: var(--x-auto);
+}
+
+.container > div > p {
+  margin-top: var(--12);
+  font-size: var(--5);
+}
+
+@media (max-width: 1024px) {
```

Comment:
Should this be one of the screens variables? Or perhaps I'm vaguely remembering that you can't use css custom properties in media queries for some reason.

Reply:
> you can't use css custom properties in media queries for some reason.

Yeah this is the reason :(
I am missing the comment though, I'll add that.